tests/services/phase.service.spec.ts

```diff
@@ -0,0 +1,114 @@
+import { of } from 'rxjs';
+import { SessionData } from '../../src/app/core/models/session.model';
+import { User, UserRole } from '../../src/app/core/models/user.model';
+import { Phase, PhaseService } from '../../src/app/core/services/phase.service';
+import { UserService } from '../../src/app/core/services/user.service';
+
+const testStudent: User  = {
+  loginId: 'testStudent',
+  role: UserRole.Student
+};
+
+const testTutor: User = {
+  loginId: 'testTutor',
+  role: UserRole.Tutor
+};
+
+const mockSettingsFile: {} = {
+  'openPhases' : [Phase.phaseBugReporting],
```

Comment:
One more minor thing 😅 The space after `openPhases` haha

Reply:
👍 psps 😅